data/wildfly-runner/src/test/java/org/hibernate/data/tck/ext/HibernateLoadableExtension.java

```diff
@@ -0,0 +1,14 @@
+package org.hibernate.data.tck.ext;
```

Comment:
This feels like an odd package name here.

Reply:
Done.